err_stash.py

```diff
@@ -549,7 +549,7 @@ def ensure_no_conflicts(stash_api, from_branch, plans):
         raise CheckError(error_lines)
 
 
-def ensure_has_pull_request(plans):
+def ensure_has_pull_request(plans):  # Already checked in `create_plans` method
```

Comment:
Add this comment because it isn't cover.

Reply:
Sure, the `raise` statement is never reached, because it is already checked in `create_plans` if has at least one PR.

Reply:
It was not well explained, this is a double check. In `create_plans`, is assured that the branches has at least one PR open, so, if doesn't happen anything unexpected, this will never raise a `CheckError`